lib/utils/messages.ts

```diff
@@ -15,26 +15,28 @@
  * limitations under the License.
  */
 
-const DEBUG_MODE = !!process.env.DEBUG;
-
 export const reachingOut = `\
-If you need help, or just want to say hi, don't hesitate in reaching out
-through our discussion and support forums at https://forums.balena.io
-
-For bug reports or feature requests, have a look at the GitHub issues or
-create a new one at: https://github.com/balena-io/balena-cli/issues/\
+For additional help or support, visit:
+https://www.balena.io/docs/reference/balena-cli/#support-faq-and-troubleshooting
 `;
 
 const debugHint = `\
 Additional information may be available with the \`--debug\` flag.
-`;
+\n`;
 
 export const help = `\
-For help, visit our support forums: https://forums.balena.io
-For bug reports or feature requests, see: https://github.com/balena-io/balena-cli/issues/
+For help, check the masterclass tutorials, FAQ and our support forums at:
+- https://www.balena.io/docs/learn/more/masterclasses/overview/
+- https://github.com/balena-io/balena-cli/blob/master/TROUBLESHOOTING.md
+- https://forums.balena.io
+
+For bug reports or feature requests, check the GitHub issues at:
+https://github.com/balena-io/balena-cli/issues/\
```

Comment:
In case of **unexpected** errors only:
With --debug:
```
$ ./bin/balena login --debug
...
You don't have sufficient privileges to run this operation.
Try running this command again prefixing it with `sudo`.

If this is not the case, and you're trying to burn an SDCard, check that the write lock is not set.

Error: EACCES: permission denied, open '/Users/paulo/.balena/token'

For help, check the masterclass tutorials, FAQ and our support forums at:
- https://www.balena.io/docs/learn/more/masterclasses/overview/
- https://github.com/balena-io/balena-cli/blob/master/TROUBLESHOOTING.md
- https://forums.balena.io

For bug reports or feature requests, check the GitHub issues at:
https://github.com/balena-io/balena-cli/issues/
```
Without --debug:
```
$ ./bin/balena login
...
You don't have sufficient privileges to run this operation.
Try running this command again prefixing it with `sudo`.

If this is not the case, and you're trying to burn an SDCard, check that the write lock is not set.

Additional information may be available with the `--debug` flag.

For help, check the masterclass tutorials, FAQ and our support forums at:
- https://www.balena.io/docs/learn/more/masterclasses/overview/
- https://github.com/balena-io/balena-cli/blob/master/TROUBLESHOOTING.md
- https://forums.balena.io

For bug reports or feature requests, check the GitHub issues at:
https://github.com/balena-io/balena-cli/issues/
```

Reply:
My main intention was to add a reference to the masterclasses somewhere. I don't mind reverting some changes in the PR if they are controversial (not worth it to reopen endless debates...). I just thought that the masterclasses are valuable, and should be "advertised" a bit more.

Reply:
Then again, it is not logical that the error message for "unexpected errors" should mention the masterclasses. I will make a change here.